Mage.Sets/src/mage/cards/c/Conspiracy.java

```diff
@@ -125,9 +126,9 @@ public boolean apply(Game game, Ability source) {
             List<Permanent> permanents = game.getState().getBattlefield().getAllActivePermanents(controller.getId());
             for (Permanent permanent : permanents) {
                 if (permanent.isCreature()) {
-                    permanent.setIsAllCreatureTypes(false);
+                    permanent.setIsAllCreatureTypes(false, game);
                     permanent.getSubtype(game).removeAll(SubType.getCreatureTypes());
```

Comment:
There is a special method in `MageObject` for that task: `removeAllCreatureTypes`

Reply:
ah yes, the method that I added